webapplication/sip/settings.py

```diff
@@ -258,3 +262,8 @@
 
 DEFAULT_SYSTEM_NOTIFICATION_EMAIL = ""
 DEFAULT_TRANSACTION_ERROR = "Something went wrong, please contact us"
+
+SENTINEL_IMAGE_NONE_TYPE = 1
+SENTINEL_IMAGE_SENTINEL1_TYPE = 2
+SENTINEL_IMAGE_SENTINEL2_TYPE = 3
+SENTINEL_IMAGE_SENTINEL1_2_TYPE = 4
```

Comment:
The view should calculate and return a set of flags. E.g. ComponentSerializer should return something like "sentinels": ["Sentinel1", "Sentinel2"].

Reply:
updated